README.md

```diff
@@ -2,112 +2,157 @@
+organizations in the Jira app.
+
+> :warning: If you are not directed, navigate to the Apps page of your Jira instance and click **Configure integration** under the ”GitHub” heading. If you do not see this, click on **Manage apps** and **Get started** under GitHub for Jira. This will bring you to the app's configuration screen.
 
-### Connecting your GitHub organization to Jira
-As part of the installation flow, you should be directed to configure your GitHub organizations in the Jira app. In your Jira instance, in the `Manage Apps` section, click on `Get Started` under GitHub for Jira.  This will bring you to the app's configuration screen.  Click `Connect GitHub organization` and follow the steps to install the app on GitHub and allow it permission to view your repos. You can also manage existing connections or add additional organizations any time on this same screen.
-![image](./docs/images/get-started.png)
+Click **Connect GitHub organization** and select the organization and repositories that you would like to connect.
 
+> :warning: If you don’t see your organization click **Install Jira on a new organization** and follow the steps to [install the app on GitHub](#install-app) and allow it permission to view your repositories. You can also manage existing connections or add additional organizations at any time on this same screen.
+
+### Connect new GitHub repositories
+If you originally gave the GitHub for Jira app access to "All repositories", and you've created a new repository on GitHub after configuration, your new repository will automatically work with the integration. However, if you installed the app on a subset of repositories, you will need to manually edit your repository selection by:
 
-#### Selecting GitHub repositories
-If you originally gave the app access to "All repositories", and you've created a new repository on GitHub after installing the GitHub integration for Jira, your new repository will automatically work with the integration. If you installed the app on a subset of repositories, you will need to manually edit your repository selection by:
 1. Sign in to your Jira Cloud account
-2. From the top navigation bar in Jira, select **Apps** -> **Manage your apps** -> **GitHub for Jira** -> **Get started**
+2. From the top navigation bar in Jira, select **Apps > Manage your apps - GitHub for Jira > Get started**.
 3. Select **Configure** next to the relevant GitHub organization and add the new repository you want the app to integrate with.
 
-### Authorization
+## Manage app
+
+### Authorize
 By granting the app access, you are providing the following authorizations to your GitHub and Jira accounts:
 
-#### Jira Permission Scopes
+#### Jira permission scopes
 Read, Write, and Admin for Development Information (branches, commits, and pull requests)
 
-#### GitHub Permission Scopes
+#### GitHub permission scopes
 
-|Permission scope|Why we need it|
+|**Permission scope**|**Why we need it**|
```

Comment:
nit: we -> the app ?

Reply:
I see you want to distance yourself from the app... noted :)